src/components/models/LogoutModel.css

```diff
@@ -0,0 +1,65 @@
+/* .logout-model{
+    display: grid;
+    grid-template-columns: auto auto auto;
+    background-color: aliceblue;
+    color: rgb(103, 4, 4);
+    text-align: center;
+    margin-left: 300px;
+    padding: 25px;
+    opacity: 20%;
+} */
+
+:root {
+    --glow-color: rgb(54, 50, 63);
+  }
+
+.logout-model {
+    display: grid; 
+    grid-template-columns: 1fr 1fr; 
+    grid-template-rows: 1fr 1fr; 
+    gap: 40px 50px; 
+    padding: 20px;
+    margin-left: 400px;
+    
+    background-color: aliceblue;
+    border-radius: 10px;
+    grid-template-areas: 
+      "LogOut LogOut"
+      "Btn-yes Btn-no"; 
+      text-align: center;
+      justify-content: center;
+      align-items: center;   
+      padding: 0.35em 3em;
+      border: 0.15em solid var(--glow-color);
+      border-radius: 0.45em;
+      
+      -webkit-box-shadow: inset 0px 0px 0.5em 0px var(--glow-color),
```

Comment:
what is the use of -webkit here?

Reply:
this box shadow effect may varies under the run platform so i mentioned it